specs/_features/maxeb_increase/capella.py

```diff
@@ -598,7 +600,6 @@ class SignedBLSToExecutionChange(Container):
     message: BLSToExecutionChange
     signature: BLSSignature
 
-
```

Comment:
No need to remove this line :)

Reply:
thanks :-) not sure how these keep getting in there.